core/src/main/java/org/apache/calcite/rel/rel2sql/SqlImplementor.java

```diff
@@ -560,17 +560,27 @@ public SqlNode toSql(RexProgram program, RexNode rex) {
         return new SqlDynamicParam(caseParam.getIndex(), POS);
 
       case IN:
-        subQuery = (RexSubQuery) rex;
-        sqlSubQuery = visitChild(0, subQuery.rel).asQueryOrValues();
-        List<RexNode> operands = subQuery.operands;
-        SqlNode op0;
-        if (operands.size() == 1) {
-          op0 = toSql(program, operands.get(0));
+        if (rex instanceof RexSubQuery) {
+          subQuery = (RexSubQuery) rex;
+          sqlSubQuery = visitChild(0, subQuery.rel).asQueryOrValues();
```

Comment:
e, can we have two separate lines here ?

Reply:
Thank you for taking a look.
I think you've probably meaned splitting it like this:
```
   sqlSubQuery =
        visitChild(0, subQuery.rel).asQueryOrValues();
```
I've updated the patch.